docs/source/platform/schema-validation.md

```diff
@@ -1,152 +1,195 @@
 ---
-title: Validating schema changes
-description: How to validate your schema in your existing CI workflow
+title: Validate schema changes
+description: Check if proposed schema changes are safe or breaking by comparing against live server traffic
 ---
 
-The Apollo GraphQL Platform allows developers to confidently iterate a GraphQL schema by validating the new schema against field-level usage data from the previous schema. By knowing exactly which clients will be broken by a new schema, developers can avoid inadvertently deploying a breaking change.
+As GraphQL scales within an organization, it becomes harder to evolve the schema while guaranteeing that no query or client will ever break from a change. Some organizations take the appraoch of just _never_ making schema changes that might be breaking; however, managing an only-ever-growing schema is unnecessarily difficult for most teams. It can actually be very safe to evolve the schema through field removals and return type changes if you have the right tools to guarantee that no such change will ever break an active query.
```

Comment:
I think the value of schema validation is defining a breaking change, so the second sentence is out of context with these new changes.

I think the current intro misses the great opportunity to speak directly to server implementors about how the complexity of a backend makes changes harder to validate. I recommend that we mention that the cli provides a high level understanding of the system that pinpoints the suspect changes and reduces on-call incidents for broken clients.